src/types/cluster.ts

```diff
@@ -0,0 +1,9 @@
+interface Cluster{
+    ID: string;
+    hatzvaraQuality: string;
+    identification: string;
+    staying: number;
+    profession: string;
+    numberOfBuildings: number;
+    geoBuildings: Array<Coordinate>;
```

Comment:
use the shorthand syntax: Coordinate[] instead of Array<Coordiante>

Reply:
fixed